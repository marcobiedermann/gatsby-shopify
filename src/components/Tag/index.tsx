import React, { FC } from 'react';

const Tag: FC = (props) => {
  return <span {...props} />;
};

export default Tag;
