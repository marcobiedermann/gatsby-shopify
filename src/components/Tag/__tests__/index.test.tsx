import { render } from '@testing-library/react';
import React from 'react';
import Tag from '..';

describe('tag component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Tag>tag</Tag>);

    expect(asFragment()).toMatchSnapshot();
  });
});
