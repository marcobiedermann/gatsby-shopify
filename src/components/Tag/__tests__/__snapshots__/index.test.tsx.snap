// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag component should render correctly 1`] = `
<DocumentFragment>
  <span>
    tag
  </span>
</DocumentFragment>
`;
